lib/adyen/client.rb

```diff
@@ -38,10 +38,8 @@ def service_url_base(service)
         "http://localhost:#{@mock_port}"
       else
         case service
-        when "Checkout"
+        when "Checkout", "CheckoutUtility"
```

Comment:
CheckoutUtility does not have checkout in name for test although it works with.
https://checkout-test.adyen.com/v1/originKeys

Reply:
@ScottPowell14 check the test cases they are failing now.